cinema_service/settings.py

```diff
@@ -20,14 +25,13 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = os.getenv("SECRET_KEY", "your-default-secret-key")
```

Comment:
Using a default secret key is not recommended for production. Ensure that the `SECRET_KEY` environment variable is set to a secure, unique value in production environments.